tests/components/ecobee/test_number.py

```diff
@@ -0,0 +1,95 @@
+"""The test for the ecobee thermostat number module."""
+from unittest import mock
+
+import pytest
+
+from homeassistant.components.ecobee import number as ecobee
+
+
+@pytest.fixture
+def ecobee_fixture():
+    """Set up ecobee mock."""
+    vals = {
+        "name": "Ecobee",
+        "identifier": "123123123",
+        "modelNumber": "athenaSmart",
+        "program": {
+            "climates": [
+                {"name": "Climate1", "climateRef": "c1"},
+                {"name": "Climate2", "climateRef": "c2"},
+            ],
+            "currentClimateRef": "c1",
+        },
+        "runtime": {
+            "connected": True,
+            "actualTemperature": 300,
+            "actualHumidity": 15,
+            "desiredHeat": 400,
+            "desiredCool": 200,
+            "desiredFanMode": "on",
+        },
+        "settings": {
+            "hvacMode": "auto",
+            "heatStages": 1,
+            "coolStages": 1,
+            "fanMinOnTime": 10,
+            "heatCoolMinDelta": 50,
+            "holdAction": "nextTransition",
+            "ventilatorType": "hrv",
+            "ventilatorMinOnTimeHome": 20,
+            "ventilatorMinOnTimeAway": 10,
+            "isVentilatorTimerOn": False,
+        },
+        "equipmentStatus": "fan",
+        "events": [
+            {
+                "name": "Event1",
+                "running": True,
+                "type": "hold",
+                "holdClimateRef": "away",
+                "endDate": "2017-01-01 10:00:00",
+                "startDate": "2017-02-02 11:00:00",
+            }
+        ],
+    }
+    mock_ecobee = mock.Mock()
+    mock_ecobee.__getitem__ = mock.Mock(side_effect=vals.__getitem__)
+    mock_ecobee.__setitem__ = mock.Mock(side_effect=vals.__setitem__)
+    return mock_ecobee
+
+
+@pytest.fixture(name="data")
+def data_fixture(ecobee_fixture):
+    """Set up data mock."""
+    data = mock.Mock()
+    data.ecobee.get_thermostat.return_value = ecobee_fixture
+    return data
+
+
+@pytest.fixture(name="number")
+def home_number_fixture(data):
```

Comment:
I think we should test by setting up the integration

Reply:
This is already tested in the [integration unit tests](https://github.com/home-assistant/core/blob/d9903c4cf985381002de8b923815b05dd24e0651/tests/components/ecobee/common.py#L11)

By adding the setting info in the fixture
```yaml
        "ventilatorType": "hrv",
        "ventilatorMinOnTimeHome": 20,
        "ventilatorMinOnTimeAway": 10,
        "isVentilatorTimerOn": false,
```
https://github.com/home-assistant/core/pull/83645/files#diff-1c4cbfd307eeb9092402c30e437b489816d22c104420d64f1e929026d82f300eR30

we do test the addition of the numbers

Reply:
ooo nevermind, I understand what you mean. 

I applied your feedback